README.md

```diff
@@ -1,10 +1,47 @@
-# Website
+# Project Layout
```

Comment:
It'd probably be good to still include an intro here explaining what the repo is for (e.g. all docs for Uniswap), that it uses Docusaurus, and generally just intro people reading!